### ISSUE TITLE
install instructions?

### ISSUE THREAD

**Issue by asanchezdelc** [hortonworks/streamline#776]
Hi there, I wanted to try this, but I dont see any instructions. I tried 
`mvn  clean install`

```
[ERROR] Failed to execute goal on project common: Could not resolve dependencies for project com.hortonworks.streamline:common:jar:0.1.0-SNAPSHOT: Could not find artifact com.hortonworks.registries:registry-common:jar:0.3.0-SNAPSHOT in clojars (https://repository.apache.org/content/repositories/snapshots/) -> [Help 1]
```

**Comment by hmcl** [comment]
@asanchezdelc was this with latest master?

**Comment by asanchezdelc** [comment]
@hmcl yeah

**Comment by HeartSaVioR** [comment]
@asanchezdelc 
Sorry for the inconvenience, seems like we miss to update instruction to build the project. 

Development branches of Streamline rely on SNAPSHOT version of Registry right now, so you need to build & install Registry artifacts locally via `mvn install` before building Streamline.

The link for Registry project is here: https://github.com/hortonworks/registry

**Comment by HeartSaVioR** [comment]
I've filed a new issue to Registry project to release 0.3.0: https://github.com/hortonworks/registry/issues/170

It would be ideal instead of guiding contributors or early adopters to build Registry project first.

**Comment by asanchezdelc** [comment]
@HeartSaVioR Thanks for the details. I was able to get a successful build. Although I have to specify a version for `streamline-runtime-storm@0.5.0.3.0.0.0-453` artifact and skip tests. 

Now what? I checked README.md history and there were some instructions to run the webservice, but it doesnt seem to work. 

```shell
java -cp target/webservice-0.1.0-SNAPSHOT.jar com.hortonworks.streamline.webservice.StreamlineApplication server conf/streamline-dev.yaml
Error: Could not find or load main class com.hortonworks.streamline.webservice.StreamlineApplication
```

**Comment by HeartSaVioR** [comment]
Yes I think documentation is totally missing. We've documented Streamline (Streaming analytics manager) for HDF 3.0.0 but missed for users/contributors who want to build and use Streamline directly.

https://docs.hortonworks.com/HDPDocuments/HDF3/HDF-3.0.0/bk_getting-started-with-stream-analytics/content/ch_building-end-to-end-app.html
https://docs.hortonworks.com/HDPDocuments/HDF3/HDF-3.0.0/bk_streaming-analytics-manager-user-guide/content/ch_sam-manage.html

Will handle this. Maybe still need to refer how to use from HDF document for some time.

**Comment by tianlin** [comment]
So can I use streamline directly without HDF？
